.github/ISSUE_TEMPLATE/release.md

```diff
@@ -12,6 +12,18 @@ assignees: ''
 These release steps can be done a week before the release, in separate PRs.
 They can be skipped for urgent releases.
 
+## State Full Sync Test
+
+To check consensus correctness, we want to test that the state format is valid after a full sync. (Format upgrades are tested in CI on each PR.)
+
+- [ ] Make sure there has been [at least one full sync test](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=event%3Aschedule) since the last state change, or
```

Comment:
```suggestion
- [ ] Make sure there has been [at least one successful full sync test](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=event%3Aschedule) since the last state change, or
```
Maybe add that there to be more clear ? right now, for example in https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=event%3Aschedule there are several failing jobs. I assume the release should be done only after one full sync pass.